sanoid

```diff
@@ -160,17 +160,17 @@ sub monitor_snapshots {
 			my $typecrit = $type . '_crit';
 			my $warn = convertTimePeriod($config{$section}{$typewarn}, $smallerperiod);
 			my $crit = convertTimePeriod($config{$section}{$typecrit}, $smallerperiod);
-			my $elapsed = -1;
+			my $elapsed;
 			if (defined $snapsbytype{$path}{$type}{'newest'}) {
 				$elapsed = $snapsbytype{$path}{$type}{'newest'};
 			}
 			my $dispelapsed = displaytime($elapsed);
 			my $dispwarn = displaytime($warn);
 			my $dispcrit = displaytime($crit);
-			if ( $elapsed > $crit || $elapsed == -1) {
+			if ($elapsed > $crit || !defined($elapsed)) {
```

Comment:
a potential undefined value is compared here and will probably trigger a warning